py-polars/tests/unit/io/test_parquet.py

```diff
@@ -417,6 +417,19 @@ def test_sink_ipc(io_files_path: Path, tmp_path: Path) -> None:
         assert_frame_equal(result, df_read)
 
 
+@pytest.mark.write_disk()
+def test_sink_csv(io_files_path: Path, tmp_path: Path) -> None:
```

Comment:
Tests for `sink` methods should go in `py-polars/tests/unit/streaming/test_streaming_io.py`

Probably good to add a few more tests, set some parameters, see if everything works as expected.

Reply:
Hi @stinodego : I added this here because the test for sink_ipc & sink_parquet are just above it...
Should I move these as well?

Reply:
Ok, I added a few tests in there.
Could you have a look? If it looks fine, I'd like to take this PR out of draft & open for review from the other maintainers.

I thank you a lot for your kind assistance in this one!